scripts/update/Updater.php

```diff
@@ -30,5 +30,9 @@ class Updater extends common_ext_ExtensionUpdater
     public function update($initialVersion)
     {
         $this->skip('1.0.0', '1.0.1');
+
+        if ($this->isVersion('1.0.1')) {
+            $this->setVersion('1.0.2');
```

Comment:
With our current approach if no additional changes you should just skip version, no if condition needed. No worries we can clean up in a next PR with another change;
```suggestion
        $this->skip('1.0.0', '1.0.2');
```

Reply:
fixed here ::
https://github.com/oat-sa/extension-remote-proctoring/pull/18/files#diff-53f22eb141a5b2bf0797704363c4aa12R32